packages/@okta/vuepress-site/docs/reference/hook-best-practices/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Hook best practices
+excerpt: >-
+  Understand the best practices at Okta to implement and secure an Okta Event Hook or Inline Hook.
+---
+
+# Hooks best practices
+
+Event Hooks and Inline Hooks are outbound calls from Okta to an external client application (external service). These outbound calls integrate custom functionality into your Okta processes. For further background information on Okta Hooks, see [Event Hooks](/docs/concepts/event-hooks) and [Inline Hooks](/docs/concepts/inline-hooks).
+
+The following sections review best practices to implement and secure Okta Event Hooks or Inline Hooks.
+
+## Securing your Hook endpoint
+
+To prevent a malicious actor from making requests to the endpoint where your Okta Hooks are sent, use the following best practices:
+
+* Configure Okta to send an Authentication header in the Hook and validate it in every request by:
+
+  * Using [HTTP Basic Authentication](/books/api-security/authn/api-authentication-options/#http-basic-authentication) in the format `base64(user:password)`. See [Adding Basic Authorization and Body Parsing](/docs/guides/common-hook-set-up-steps/) for an example implementation of Basic Authentication.
```

Comment:
Did you mean to use this link for Adding Basic Authorization and Body Parsing?
/docs/guides/common-hook-set-up-steps/nodejs/adding-basic-auth/

I don't think this page above is a great page as it is pretty specific to setting up Basic Auth in the Glitch project. We should really spell out out to setup Basic Auth in Hooks on the actual page. For example, something like:

To use Basic Auth, set Authentication field to "Authorization" and Authentication secret field to the Base64 encoded-string of "username:password:.

Reply:
Also, can we show an example snippet of what the header would look like such as:

    Accept: application/json
    Content-Type: application/json
    Authorization: Basic BASE64_ENCODED_USERNAME_PASSWORD